.gitignore

```diff
@@ -0,0 +1,112 @@
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio
+
+*.iml
+
+## Directory-based project format:
+.idea/
+# if you remove the above rule, at least ignore the following:
+
+# User-specific stuff:
```

Comment:
@mDibyo could we remove the commented out lines that don't provide any value / info?